src/simulate/event_transformer/Transformer.py

```diff
@@ -0,0 +1,80 @@
+import logging
+import utils
+import random
+import json
+
+
+class Transformer:
+    CONST_USER_STR = "https://mcommunity.umich.edu/#profile:"
```

Comment:
Minor suggestion: You don't need to mark constants with "CONST_".  Python  (and some other languages) commonly use all upper-case variables as constants.

Reply:
Done